recepten.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Gezonde Gecepten</title>
+</head>
+<body>
+    <h1>Gezonde Recepten</h1>
+    <p>doe jij maar pannenkoeken maken</p>
```

Comment:
Ik weet nooit of het pannekoek of pannenkoek is.

Reply:
pannenkoek